setup/command/list.go

```diff
@@ -58,7 +59,7 @@ func (f *CommandFactory) List() cli.Command {
 }
 
 func (f *CommandFactory) addRemoteInstances(c *cli.Context, current map[string]status) error {
-	provider, err := f.newAWSProviderHelper(c)
+	provider, err := f.newAWSProviderHelper(c, aws.DEFAULT_AWS_REGION)
```

Comment:
[nit] I'm mildly confused about using the name `DEFAULT_AWS_REGION` at this point --- is that really what this is?

Reply:
So my thinking here is that `l0-setup list` is sort of a weird case. We need the AWS Provider, but the region doesn't matter at all because all list does when it checks for remote instances is it checks S3 (which is region agnostic for lists) and finds buckets that correlate to l0 instances. So because I modified the helper function, I need to pass in some sort of region, but the region doesn't matter, so I opt for the default region.

Reply:
can you pass in `nil` for the region, then? My concern here is that the code is a little misleading without the context you just gave.

Reply:
The problem with sending `nil` is that I use `config.WithRegion(region)`, and the `aws.Config` struct has a field called Region which is required:
https://github.com/aws/aws-sdk-go/blob/master/aws/config.go#L70

So I have to pass a valid string (and a valid region) into `WithRegion()`